Hymns/Display/SoundCloudView.swift

```diff
@@ -0,0 +1,44 @@
+import SwiftUI
+
+struct SoundCloudView: View {
+    @State private var isLoading = true
+    @Binding var showSoundCloud: Bool
+    @Binding var soundCloudinitiated: Bool
```

Comment:
If we are just looking at `isOtherAudioPlaying` then we don't need this right?

Reply:
I think we still do. The reason why is we need a way to actually close the webView and stop the music. So let's say that the user open SoundCloud, plays music, goes back to the lyrics. Music is playing and then they want to stop that music from playing. So they click our X on the banner. We need to be able to close out that webView.   

That is what this var does